arangod/RestServer/DatabaseFeature.h

```diff
@@ -23,25 +23,42 @@
 #ifndef APPLICATION_FEATURES_DATABASE_FEATURE_H
 #define APPLICATION_FEATURES_DATABASE_FEATURE_H 1
 
+#if !defined(USE_CATCH_TESTS) && !defined(EXPAND_APPLICATION_FEATURES_DATABASE_FEATURE_H)
+  #define DO_EXPAND_APPLICATION_FEATURES_DATABASE_FEATURE_H(VAL) VAL ## 1
+  #define EXPAND_APPLICATION_FEATURES_DATABASE_FEATURE_H(VAL) DO_EXPAND_APPLICATION_FEATURES_DATABASE_FEATURE_H(VAL)
+  #if defined(TEST_VIRTUAL) && (EXPAND_APPLICATION_FEATURES_DATABASE_FEATURE_H(TEST_VIRTUAL) != 1)
+    #define USE_CATCH_TESTS
+  #endif
+  #undef EXPAND_APPLICATION_FEATURES_DATABASE_FEATURE_H
+  #undef DO_EXPAND_APPLICATION_FEATURES_DATABASE_FEATURE_H
+#endif
```

Comment:
why isn' this removed ?

Reply:
I was planning to remove this in the next pull request once I have a chance to update my local workspace